Cargo.lock

```diff
@@ -1181,9 +1411,9 @@ checksum = "b7e5500299e16ebb147ae15a00a942af264cf3688f47923b8fc2cd5858f23ad3"
 
 [[package]]
 name = "openssl"
-version = "0.10.48"
```

Comment:
This file has more changes than I would have expected. In particular I think this change is causing the CI failure.

One thing to note is that if you delete the Cargo.lock file, it will get fully regenerated with new versions. To just diff based on new dependencies, you can checkout the Cargo.lock file from main and then run `cargo build` in the root.

Reply:
I think you'll want to copy the Cargo.lock file from main and then run `cargo build`. Instead of deleting it and starting from scratch.